schema/data_feed/tracker.fbs

```diff
@@ -58,4 +61,10 @@ table TrackerInfo {
 
     /// name to display as the name of the tracker set by the user
     custom_name: string;
+    
+    // Reference adjustment parameters for IMU trackers (full reset, yaw, mounting)
```

Comment:
`///` style comments become documentation on generated code, so we generally prefer that for docstrings.
```suggestion
    /// Reference adjustment parameters for IMU trackers (full reset, yaw, mounting)
```